lib/sles4sap/sdaf_deployment_library.pm

```diff
@@ -54,15 +54,90 @@ our @EXPORT = qw(
   sdaf_get_deployer_ip
   serial_console_diag_banner
   set_common_sdaf_os_env
-  prepare_sdaf_repo
+  prepare_sdaf_project
   generate_resource_group_name
   set_os_variable
+  get_os_variable
   prepare_tfvars_file
   sdaf_execute_deployment
   load_os_env_variables
   sdaf_cleanup
+  convert_region_to_long
+  convert_region_to_short
 );
 
+# SDAF uses own internal 4 character abbreviations for PC region names
+# This is a matrix used for translation.  It contains only commonly used regions, if you need to extend the list
+# you can find definitions in the function B<get_region_code> located in sdaf shell script:
+# https://github.com/Azure/sap-automation/blob/3c5d0d882f5892ae2159e262062e29c2b3fe59d9/deploy/scripts/deploy_utils.sh#L403
```

Comment:
Couldn't we call a/the SDAF shell script from the test code and get the transalation?

Reply:
Hmm... I think the function is being called before cloning repository. 
But in real life we will be probably using only one region so I am not sure if it is worth the effort.